documentation/general/design/images/hld/~$dash-hld-project-ecosystem.svg.bkp

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Looks like a backup file. You could make a `.gitignore` file containing `*.bkp` anywhere in the file hierarchy, e.g. under `Documentation/` to avoid checking in these kinds of scratch files.

Reply:
Thank you for catching this. I removed the *.bkp artifacts and created a related `.ignore` file.